src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -409,23 +408,31 @@ def generate_single_bead_permutations(
         material_name : str
             Material name.
         laser_powers : list[float]
-            Laser powers (W) to use for single bead simulations.
+            Laser powers (W) to use for single bead simulations. Valid values
```

Comment:
Looks like I failed you yesterday in understanding the values for these constants would be used. Nice job!